scalableminds/webknossos
# Add switch orga to legacy routes

Follow up bug introduced by #8075. In #8075 we missed to support the switch orga functionality for legacy routes. This pr add this functionally now.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- I suggest testing locally as it might be easier
- In one browser window (e.g. default window) open your local wk with sample user
- turn on `isWkOrgInstance` in `application.conf`
- in another browser window with a new session ( e.g. private window) create a new orga
- in that orga upload some dataset
- after successful upload open the edit view of the ds
- open the advanced tab in the data settings. On top of this, the orgaId/directoryPath combination should be listed. e.g.: `Dataset Configuration (as stored on datastore localhost at 15ca057a580b27dc/png_rgb)`. Copy the `orgaId/directoryPath` part.
- switch back to the window where the sample (!super user!) is logged in. Now open `<wk>/datasets/<orgaId/directoryPath>/view`. The page should automatically redirect to `<wk>/datasets/<name-datasetId>/view` and then fail. The user should be prompted to switch the organization. Upon switching the DS should load

### Issues:
- fixes bug introduced by #8075
- https://scm.slack.com/archives/CMBMU5684/p1733217497599509?thread_ts=1733194567.434879&cid=CMBMU5684
- also fixes #8237

------
(Please delete unneeded items, merge only when none are left open)
